opt-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+
+services:
+
+  postgres_db:
+    image: postgres
+    restart: always
+
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=school
+      - POSTGRES_DB=school
+
+    volumes:
+      - ../../data/postgres:/var/lib/postgresql/data
+      - ../../data/dump:/db-dump
+
+    ports:
+      - "5432:5432"
+
+    networks:
+      - docker-practice-net
+
+
+  redis:
+    image: "redis:alpine"
+    command: redis-server
+
+    ports:
+      - "6379:6379"
+
+    environment:
+      - REDIS_REPLICATION_MODE=master
```

Comment:
This is probably the same as for the 27th line - does not basic Redis image works without this environment? Or this is kind of some enhancement? Could you please explain.

Reply:
Again, I did as in the example, apparently not the best found, I do not know.